### PR TITLE
[MUMPS_seq] 5.4.1 + build on experimental platforms

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

**Comment by odow** [comment]
I think I'm out of my depth on this one.

**Comment by giordano** [comment]
The problem in PowerPC
```
[00:18:25] gfortran -fPIC -o csimpletest -O   csimpletest.o  ../lib/libcmumps.a ../lib/libmumps_common.a -L/workspace/destdir/lib -lmetis -L../PORD/lib/ -lpord  -L/workspace/destdir/lib -lopenblas -L../libseq -lmpiseq -L/workspace/destdir/lib -lopenblas -lpthread
[00:18:25] /workspace/destdir/lib/libopenblas.so: undefined reference to `GOMP_parallel@GOMP_4.0'
[00:18:25] collect2: error: ld returned 1 exit status
```
is the same as https://github.com/JuliaPackaging/Yggdrasil/pull/368#discussion_r372541291.  We need to use at least GCC 5.  I don't remember whether we have something wrong in our compiler shard or it's an upstream bug

File: M/MUMPS/MUMPS_seq@5/build_tarballs.jl [hunk 1]
```diff
@@ -91,4 +100,4 @@ dependencies = [
 ]
 
 # Build the tarballs
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat = "1.6")
```
Comment (odow) [inline M/MUMPS/MUMPS_seq@5/build_tarballs.jl:103]: Can I just exclude the platform? Or should I set a gcc_version instead?

Comment (giordano) [reply M/MUMPS/MUMPS_seq@5/build_tarballs.jl:103]: I think doing
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat = "1.6", preferred_gcc_version=v"5")
```
is simpler than excluding a platform which would make Valentin sad :stuck_out_tongue:

**Pull request merged**